### PR TITLE
Query of proposal

### PR DESCRIPTION
**Pull request by ahmedalmi95**
query for getting proposals

### CODE REVIEW

**Review by Anies12 — APPROVED** [review]

**Review by Fatmasiam — APPROVED** [review]

File: src/database/queries/getData.js [hunk 1]
```diff
@@ -24,6 +24,12 @@ const getSpecalize = (categoryId) => {
   };
   return connect.query(sql);
 };
+const getProposals = (specId) => {
+  const sql = 'select proposal.title,  proposal.description, proposal.id, users.firstname,users.lastname,users.photo_url  from proposal inner join users on proposal.user_id = users.id where proposal.specalization_id = $1';
+  const values = [specId];
+  return connect.query(sql, values);
```
Comment (tomduggan85) [inline src/database/queries/getData.js:36]: @AhmedAl-Almi This looks good! My only suggestion is to add spaces after the user fields in the query: `... users.firstname, users.lastname, users.photo_url from proposal ...` to make it a little easier to read.

Comment (ahmedalmi95) [reply src/database/queries/getData.js:36]: Sure, thanks Tom

**Review by Anies12 — APPROVED** [review]

**Review by Angham116 — APPROVED** [review]

**Comment by kannanhassouna** [comment]
@AhmedAl-Almi , where the description of the P.R

File: test/test.js [hunk 2]
```diff
@@ -326,6 +326,20 @@ tape('test add user for mobile number', (t) => {
...
+tape('Test Home route ', (t) => {
+  supertest(router)
+    .get('/')
+    .expect(200)
+    .expect('content-type', /html/)
+    .end((error, res) => {
+      if (error) {
+        t.error(error);
+      }
+      t.equal(typeof res.body, 'object', 'should return type of body object');
```
Comment (kannanhassouna) [inline test/test.js]: really, man where the test of the query?
and what i will get when i check the type of the req.body.

###  MAKE TESTS

File: src/database/queries/getData.js [hunk 3]
```diff
@@ -13,17 +13,26 @@ const checkEmail = (email) => {
...
   const sql = {
-    text: 'select id , name  from specialization where field_id = $1',
+    text: 'select id, name from specialization where field_id = $1',
     values: [categoryId],
   };
   return connect.query(sql);
 };
+
+const getProposals = (specId) => {
+  const sql = 'select proposal.title, proposal.description, proposal.id, users.firstname, users.lastname, users.photo_url from proposal inner join users on proposal.user_id = users.id where proposal.specalization_id = $1';
```
Comment (kannanhassouna) [inline src/database/queries/getData.js]: make your query readable like 
```suggestion
 
  const sql =
"select proposal.title, proposal.description, proposal.id, users.firstname, users.lastname, users.photo_url " +
  "from proposal inner join users" +
  "on proposal.user_id = users.id" +
  "where proposal.specalization_id = $1";
  ```

**Review by kannanhassouna — CHANGES_REQUESTED** [review]
i need actual tests

File: test/test.js (hunk 2 above)
Comment (Fatmasiam) [reply test/test.js]: @knanahassouna1  done :+1:

File: src/database/queries/getData.js (hunk 3 above)
Comment (Fatmasiam) [reply src/database/queries/getData.js]: @knanahassouna1  done :+1:

**Pull request merged**